docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1493,9 +1506,20 @@ These are not currently modeled.
     - 
   * - Age 
     -  
-  * - Social Security Number 
+  * - Social Security Number or ITIN
     -
 
+If a simulant does not have a SSN but is filing taxes, please 
+include an Individual Taxpayer Identification Number (ITIN) instead. 
```

Comment:
Will you want this to be the same for a given simulant across years? If so, you might want to add a section about initializing it, separate from the tax observer, and just reference it here.

(Sorry, I initially put this comment on the commit instead of the PR.)

Reply:
Hmm we do want to track over time, but I am imagining that this is assigned on time steps, not initialized for anyone. It will just apply on as "as needed" basis. For that - should I still add it to another section? Or just here with some more written about how we want it tracked?

Reply:
Hm... you mean that someone would only be assigned an ITIN when they are actually observed filing taxes? That makes sense but I am not sure if observers assigning values is something we do in Vivarium. That's why I had assumed it would be assigned upfront and just _used_ by the observer.

Reply:
Yeah, exactly, you just assign the number when the observer needs it. @beatrixkh can you weigh in? I can initialize this for everyone if needed.

Reply:
sorry for the delay! yes, if someone should have the same ITN over time, we'll want to initialize it with them as zeb is suggesting. then if they never use it you'll never see it, but if they use it, it's permanently associated with them and won't change from one timestep to another.

and yes --- also, as zeb was thinking, observers are not able to effect the state of anything in the simulation (such as a simulant's id, age, etc.), they are only looking at the current state of the simulation, potentially doing some processing that doesn't change the state of the simulation, and then outputting those observations.

Reply:
Thanks! I am going to change this to be randomly assigned for now. If it proves to be important for PRL, we can add this as a new initialized concept later.